src/components/TransactionSentDialog/index.js

```diff
@@ -45,14 +44,15 @@ export default class TransactionSentDialog extends Component {
   }
 
   static defaultProps = {
-    txReturn: {},
+    txReturn: undefined,
   }
 
   render() {
     const { intl, classes, txReturn, clearTxReturn } = this.props;
+    if (!txReturn) return null;
```

Comment:
This is the line that needs to get removed so that when a transaction happens, this dialog will display. Without this line, line 59 will error. `cannot access txid of undefined` because it's not an object.
We can do a quick fix to this by just making the default in the component be `{}`, and the global one be `undefined`

Reply:
tested already with multiple txs and is working.